rattle_cli/rattle_cli.py

```diff
@@ -76,13 +76,18 @@ def main():
     parser.add_argument("--details",
                         help="Also show the book details for each language",
                         action="store_true")
+    parser.add_argument("--shelf",
+                        help="One of your exclusive shelves. By default; \
+                        read, currently-reading, to-read",
```

Comment:
For clarity, I think I would include in the help string that it defaults to "read" when unspecified.

Reply:
Let me know if my update is clear enough.